### DISCUSSION TITLE
Icons sometimes show as unicode jumble

### DISCUSSION THREAD

**Discussion by Mhbuur** [microsoft/fluentui#30145] [Q&A]
I'm facing a rather strange problem with fluent ui icons sometimes not showing correctly in my SPA:
![image](https://github.com/microsoft/fluentui/assets/419020/55e42705-0e49-4a09-8d12-83201ab17de7)

If I reload the application, the icons usually show correctly again (though once in a while the unicode jumble sticks for several reloads):
![image](https://github.com/microsoft/fluentui/assets/419020/88e8890d-5cfd-46c3-afac-76144dff0fdf)

When my app starts, I'm calling 
`initializeIcons(undefined, { disableWarnings: true });`

I'm not seeing any console or network errors. 

I would greatly appreciate any insights, suggestions, or guidance on potential causes and where I should focus my attention to pinpoint and rectify the problem.

**Comment by Mhbuur** [comment]
This problem is still haunting me. 

I tried blocking CDN requests completely to see if the behaviour is related to missing font files etc.
This causes the rendering to look like this:
![image](https://github.com/microsoft/fluentui/assets/419020/6843a60d-dbf0-4a9c-856b-686d3852e761)

This is a common "unknown character" placeholder and makes sense because the resources were blocked. 
The initial problem with the unicode jumble is unlikely to be related to missing font files.

Rather it seems to be some fundamental unicode parsing problem. Almost as if the document temporary looses its encoding information, which is not the case.

Again, any help or theories is greatly appreciated. No console errors, no network errors, no service worker interference.. just no icons 🤷‍♂️:
![image](https://github.com/microsoft/fluentui/assets/419020/1d5ee36e-12dc-4062-b8ea-042509eef96d)

**Comment by FlowIT-JIT** [comment]
I have observed the same behaviour in Teams. It might have been resolved in the most recent "New" version though:
<img width="284" alt="image" src="https://github.com/microsoft/fluentui/assets/10479064/95a6290d-af26-4b21-a4b4-47cf7b30600c">

**Comment by Mhbuur** [comment]
This appears to be the same issue as discussed here:

https://github.com/microsoft/fluentui/issues/27777
and here
https://github.com/microsoft/fluentui/issues/28674

It seems like I have fixed this by adding `@charset "UTF-8";` as the very first thing in my App.scss file
